screens/QuestionScreen.js

```diff
@@ -51,12 +54,13 @@ const QuestionScreen = ({ navigation }) => {
     //Grabs the specific question from api
     const fetchData = async () => {
       axios
-        .get(apiConfig.baseUrl + "questions/1", {
+        .get(apiConfig.baseUrl + "questions/1/", {
```

Comment:
I see the tweak here (final slash) but the hardcoded URL in general is to be replaced. I believe we have a ticket in this sprint that will cover that (CA-3).